runtime/compiler/runtime/IProfiler.cpp

```diff
@@ -3147,9 +3147,9 @@ TR_IPBCDataCallGraph::loadFromPersistentCopy(TR_IPBCDataStorageHeader * storage,
          {
          J9Class *ramClass = NULL;
 
-         uintptrj_t csInfoClazz = store->_csInfo.getClazz(i);
-         if (comp->fej9()->sharedCache()->isPointerInSharedCache((void *)csInfoClazz))
-            ramClass = ((TR_J9VM *)comp->fej9())->matchRAMclassFromROMclass((J9ROMClass *)comp->fej9()->sharedCache()->pointerFromOffsetInSharedCache(csInfoClazz), comp);
+         uintptrj_t csInfoClazzOffset = store->_csInfo.getClazz(i);
+         if (comp->fej9()->sharedCache()->isOffsetInSharedCache(csInfoClazzOffset))
+            ramClass = ((TR_J9VM *)comp->fej9())->matchRAMclassFromROMclass((J9ROMClass *)comp->fej9()->sharedCache()->pointerFromOffsetInSharedCache(csInfoClazzOffset), comp);
```

Comment:
Can I suggest getting the ROM class pointer by doing `isOffsetInSharedCache(csInfoClazzOffset, &romClass)` instead? It would be a bit more optimal.